<ui version="4.0" >
 <class>NeuerUserDialogClass</class>
 <widget class="QDialog" name="NeuerUserDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NeuerUserDialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Neuer Benutzer</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Benutzername:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="usernameLineEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Name / Vorname:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="nameLineEdit" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Passwort:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="pwdLineEdit" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Passwort wiederholen:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="pwdwLineEdit" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="emailLineEdit" />
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Typ:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QComboBox" name="typComboBox" />
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Berechtigungen:</string>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="6" column="1" >
       <widget class="QTreeWidget" name="streckenTreeWidget" >
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>(mehrfachauswahl m√∂glich)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>158</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="canButton" >
       <property name="text" >
        <string>Abbrechen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
